src/main/resources/biz_de.properties

```diff
@@ -211,6 +211,8 @@ FileImportJob.importingZipFile = Die Datei wird für den Import entpackt.
 FileImportJob.importingZippedFile = Die entpackte Datei '${filename}' wird importiert.
 FileImportJob.noZippedFileFound = Das Archiv enthält keine unterstützte Datei.
 FileImportJobFactory.file = Datei
+LineBasedImportExportJobFactory.outputFile = Ausgabedatei
```

Comment:
Englische Übersetzungen?

Reply:
mach mer in lokalise - i18n ist keiner entwicklerarbeit, das ist zu teuer